### DISCUSSION TITLE
How to give 2 condition in where clause [Apr 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5485] [Discord Threads]
> 🙋‍♂️ **Rahim Dobani**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Refer following documentation : https://docs.nocodb.com/developer-resources/rest-apis#query-params   
####    
####    
#### As an example you can pass `where` query parameter like `(colName,eq,colValue)~or(colName2,gt,colValue2)`  
#  
> 👨‍💻 **pranavxc**  
#### https://docs.nocodb.com/developer-resources/rest-apis#logical-operators  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1083050261209174047)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
